CONTRIBUTING.md

```diff
@@ -7,6 +7,23 @@ follow the instructions provided in [Microsoft Azure Projects Contribution Guide
 
 If you want to contribute to a file that is generated (header contains `Code generated by Microsoft (R) AutoRest Code Generator.`), the best approach to open a PR on the initial Swagger specification, as we can NOT merge a PR on generated code (it would be replaced by next generation). See https://github.com/Azure/azure-rest-api-specs/ for details.
 
+## Live Testing Prerequisites
```

Comment:
Should we update https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md instead ?

Reply:
Agree that this should be in `tests.md` under the `Run and record the test` section

Reply:
Sounds good. I'll move it there.